make static is spitting out a bunch of warnings

Doesn't seem right.

```
cd $SRCDIR; make static
mkdir -p client/build
docker run -ti -v /home/ubuntu/src/github.com/weaveworks/scope/client/app:/home/weave/app \
		-v /home/ubuntu/src/github.com/weaveworks/scope/client/build:/home/weave/build \
		weaveworks/scope-ui-build gulp build --release
[12:01:06] Using gulpfile /home/weave/gulpfile.js
[12:01:06] Starting 'clean'...
[12:01:06] Finished 'clean' after 7.5 ms
[12:01:06] Starting 'build'...
[12:01:06] Starting 'vendor'...
[12:01:06] Starting 'html'...
[12:01:07] Starting 'favicon'...
[12:01:07] Starting 'bundle'...
[12:01:07] Finished 'favicon' after 540 ms
[12:01:08] 'html' all files 5.03 kB
[12:01:08] Finished 'html' after 1.84 s
[12:01:12] Finished 'vendor' after 6 s

WARNING in (undefined) ./app/scripts/stores/app-store.js

/home/weave/app/scripts/stores/app-store.js
  177:6  warning  Unexpected console statement  no-console

✖ 1 problem (0 errors, 1 warning)


WARNING in (undefined) ./app/scripts/charts/nodes-chart.js

/home/weave/app/scripts/charts/nodes-chart.js
  166:12  warning  Unexpected console statement  no-console
  192:4   warning  Unexpected console statement  no-console
  201:4   warning  Unexpected console statement  no-console

✖ 3 problems (0 errors, 3 warnings)


WARNING in (undefined) ./app/scripts/charts/nodes-layout.js

/home/weave/app/scripts/charts/nodes-layout.js
  12:4  warning  Unexpected console statement  no-console

✖ 1 problem (0 errors, 1 warning)


WARNING in (undefined) ./~/dagre/lib/graphlib.js
Critical dependencies:
5:4-11 require function is used in a way in which dependencies cannot be statically extracted
 @ ./~/dagre/lib/graphlib.js 5:4-11

WARNING in (undefined) ./~/dagre/lib/lodash.js
Critical dependencies:
5:4-11 require function is used in a way in which dependencies cannot be statically extracted
 @ ./~/dagre/lib/lodash.js 5:4-11

WARNING in (undefined) app.js from UglifyJs
Side effects in initialization of unused variable className [./~/material-ui/lib/js/paper.js:31,0]
Side effects in initialization of unused variable circle [./~/material-ui/lib/js/paper.js:31,0]
Side effects in initialization of unused variable innerClassName [./~/material-ui/lib/js/paper.js:31,0]
Side effects in initialization of unused variable rounded [./~/material-ui/lib/js/paper.js:31,0]
Side effects in initialization of unused variable zDepth [./~/material-ui/lib/js/paper.js:31,0]
Condition always true [./~/dagre/~/graphlib/lib/lodash.js:5,0]
Side effects in initialization of unused variable support [./~/lodash/index.js:4351,0]
Dropping unused variable object [./~/lodash/index.js:948,0]
Condition always true [./~/lodash/index.js:12207,0]
Dropping unreachable code [./~/lodash/index.js:12221,0]
Side effects in initialization of unused variable moduleExports [./~/lodash/index.js:275,0]
Side effects in initialization of unused variable onBlur [./~/material-ui/lib/js/enhanced-button.js:46,0]
Side effects in initialization of unused variable onFocus [./~/material-ui/lib/js/enhanced-button.js:46,0]
Side effects in initialization of unused variable onTouchTap [./~/material-ui/lib/js/enhanced-button.js:46,0]
Side effects in initialization of unused variable className [./~/material-ui/lib/js/font-icon.js:13,0]
Side effects in initialization of unused variable hasNewDefaultProp [./~/material-ui/lib/js/enhanced-switch.js:59,0]
Side effects in initialization of unused variable type [./~/material-ui/lib/js/enhanced-switch.js:94,0]
Side effects in initialization of unused variable name [./~/material-ui/lib/js/enhanced-switch.js:94,0]
Side effects in initialization of unused variable value [./~/material-ui/lib/js/enhanced-switch.js:94,0]
Side effects in initialization of unused variable label [./~/material-ui/lib/js/enhanced-switch.js:94,0]
Side effects in initialization of unused variable onSwitch [./~/material-ui/lib/js/enhanced-switch.js:94,0]
Side effects in initialization of unused variable defaultSwitched [./~/material-ui/lib/js/enhanced-switch.js:94,0]
Side effects in initialization of unused variable onBlur [./~/material-ui/lib/js/enhanced-switch.js:94,0]
Side effects in initialization of unused variable onFocus [./~/material-ui/lib/js/enhanced-switch.js:94,0]
Side effects in initialization of unused variable onMouseUp [./~/material-ui/lib/js/enhanced-switch.js:94,0]
Side effects in initialization of unused variable onMouseDown [./~/material-ui/lib/js/enhanced-switch.js:94,0]
Side effects in initialization of unused variable onMouseOut [./~/material-ui/lib/js/enhanced-switch.js:94,0]
Side effects in initialization of unused variable onTouchStart [./~/material-ui/lib/js/enhanced-switch.js:94,0]
Side effects in initialization of unused variable onTouchEnd [./~/material-ui/lib/js/enhanced-switch.js:94,0]
Side effects in initialization of unused variable icon [./~/material-ui/lib/js/menu/menu.js:195,0]
Side effects in initialization of unused variable data [./~/material-ui/lib/js/menu/menu.js:195,0]
Side effects in initialization of unused variable attribute [./~/material-ui/lib/js/menu/menu.js:195,0]
Side effects in initialization of unused variable number [./~/material-ui/lib/js/menu/menu.js:195,0]
Side effects in initialization of unused variable toggle [./~/material-ui/lib/js/menu/menu.js:195,0]
Side effects in initialization of unused variable onClick [./~/material-ui/lib/js/menu/menu.js:195,0]
Condition always true [./~/d3/d3.js:9502,0]
Dropping unreachable code [./~/d3/d3.js:9502,61]
Side effects in initialization of unused variable className [./~/material-ui/lib/js/transition-groups/slide-in.js:24,0]
Side effects in initialization of unused variable direction [./~/material-ui/lib/js/transition-groups/slide-in.js:24,0]
Dropping unused variable isCurryBound [./~/dagre/~/lodash/dist/lodash.compat.js:1674,0]
Condition always true [./~/dagre/~/lodash/dist/lodash.compat.js:7131,0]
Dropping unreachable code [./~/dagre/~/lodash/dist/lodash.compat.js:7144,0]
Side effects in initialization of unused variable moduleExports [./~/dagre/~/lodash/dist/lodash.compat.js:175,0]
Side effects in initialization of unused variable onClick [./~/material-ui/lib/js/menu/menu-item.js:68,0]
Side effects in initialization of unused variable onToggle [./~/material-ui/lib/js/menu/menu-item.js:68,0]
Side effects in initialization of unused variable children [./~/material-ui/lib/js/menu/menu-item.js:68,0]
Side effects in initialization of unused variable label [./~/material-ui/lib/js/menu/menu-item.js:68,0]
Side effects in initialization of unused variable className [./~/material-ui/lib/js/overlay.js:28,0]
Side effects in initialization of unused variable onCheck [./~/material-ui/lib/js/radio-button.js:20,0]
Side effects in initialization of unused variable TabTemplate [./~/material-ui/lib/js/tabs/tab.js:3,0]
Side effects in initialization of unused variable className [./~/material-ui/lib/js/text-field.js:70,0]
Side effects in initialization of unused variable errorText [./~/material-ui/lib/js/text-field.js:70,0]
Side effects in initialization of unused variable floatingLabelText [./~/material-ui/lib/js/text-field.js:70,0]
Side effects in initialization of unused variable hintText [./~/material-ui/lib/js/text-field.js:70,0]
Side effects in initialization of unused variable id [./~/material-ui/lib/js/text-field.js:70,0]
Side effects in initialization of unused variable multiLine [./~/material-ui/lib/js/text-field.js:70,0]
Side effects in initialization of unused variable onBlur [./~/material-ui/lib/js/text-field.js:70,0]
Side effects in initialization of unused variable onChange [./~/material-ui/lib/js/text-field.js:70,0]
Side effects in initialization of unused variable onFocus [./~/material-ui/lib/js/text-field.js:70,0]
Side effects in initialization of unused variable type [./~/material-ui/lib/js/text-field.js:70,0]
Side effects in initialization of unused variable onToggle [./~/material-ui/lib/js/toggle.js:20,0]
Side effects in initialization of unused variable className [./~/material-ui/lib/js/tooltip.js:27,0]
Side effects in initialization of unused variable label [./~/material-ui/lib/js/tooltip.js:27,0]
Condition always true [./~/material-ui/~/classnames/index.js:39,0]
Condition always false [./~/flux/lib/invariant.js:26,0]
Dropping unreachable code [./~/flux/lib/invariant.js:27,0]
Condition always false [./~/font-awesome-webpack/~/style-loader/addStyles.js:23,0]
Dropping unreachable code [./~/font-awesome-webpack/~/style-loader/addStyles.js:24,0]
Condition always false [./~/font-awesome-webpack/~/style-loader!./~/font-awesome-webpack/~/css-loader!./~/font-awesome-webpack/~/less-loader!./~/font-awesome-webpack/font-awesome-styles.loader.js!./~/font-awesome-webpack/font-awesome.config.js:9,0]
Dropping unreachable code [./~/font-awesome-webpack/~/style-loader!./~/font-awesome-webpack/~/css-loader!./~/font-awesome-webpack/~/less-loader!./~/font-awesome-webpack/font-awesome-styles.loader.js!./~/font-awesome-webpack/font-awesome.config.js:11,0]
Side effects in initialization of unused variable update [./~/font-awesome-webpack/~/style-loader!./~/font-awesome-webpack/~/css-loader!./~/font-awesome-webpack/~/less-loader!./~/font-awesome-webpack/font-awesome-styles.loader.js!./~/font-awesome-webpack/font-awesome.config.js:7,0]
Side effects in initialization of unused variable onTouchTap [./~/material-ui/lib/js/app-bar.js:42,0]
Side effects in initialization of unused variable other [./~/material-ui/lib/js/app-bar.js:42,0]
Side effects in initialization of unused variable onCheck [./~/material-ui/lib/js/checkbox.js:19,0]
Dropping unused variable newSelectedDate [./~/material-ui/lib/js/date-picker/calendar.js:159,0]
Side effects in initialization of unused variable selectedDate [./~/material-ui/lib/js/date-picker/date-display.js:35,0]
Side effects in initialization of unused variable initialDate [./~/material-ui/lib/js/date-picker/date-picker-dialog.js:37,0]
Side effects in initialization of unused variable onAccept [./~/material-ui/lib/js/date-picker/date-picker-dialog.js:37,0]
Side effects in initialization of unused variable formatDate [./~/material-ui/lib/js/date-picker/date-picker.js:59,0]
Side effects in initialization of unused variable mode [./~/material-ui/lib/js/date-picker/date-picker.js:59,0]
Side effects in initialization of unused variable onFocus [./~/material-ui/lib/js/date-picker/date-picker.js:59,0]
Side effects in initialization of unused variable onTouchTap [./~/material-ui/lib/js/date-picker/date-picker.js:59,0]
Side effects in initialization of unused variable KeyCode [./~/material-ui/lib/js/date-picker/date-picker.js:5,0]
Side effects in initialization of unused variable className [./~/material-ui/lib/js/date-picker/day-button.js:23,0]
Side effects in initialization of unused variable date [./~/material-ui/lib/js/date-picker/day-button.js:23,0]
Side effects in initialization of unused variable onTouchTap [./~/material-ui/lib/js/date-picker/day-button.js:23,0]
Side effects in initialization of unused variable selected [./~/material-ui/lib/js/date-picker/day-button.js:23,0]
Side effects in initialization of unused variable className [./~/material-ui/lib/js/dialog.js:17,0]
Side effects in initialization of unused variable KeyLine [./~/material-ui/lib/js/drop-down-icon.js:4,0]
Side effects in initialization of unused variable Paper [./~/material-ui/lib/js/drop-down-icon.js:5,0]
Side effects in initialization of unused variable className [./~/material-ui/lib/js/enhanced-textarea.js:41,0]
Side effects in initialization of unused variable onChange [./~/material-ui/lib/js/enhanced-textarea.js:41,0]
Side effects in initialization of unused variable onHeightChange [./~/material-ui/lib/js/enhanced-textarea.js:41,0]
Side effects in initialization of unused variable rows [./~/material-ui/lib/js/enhanced-textarea.js:41,0]
Side effects in initialization of unused variable valueLink [./~/material-ui/lib/js/enhanced-textarea.js:41,0]
Side effects in initialization of unused variable name [./~/material-ui/lib/js/radio-button-group.js:57,0]
Side effects in initialization of unused variable value [./~/material-ui/lib/js/radio-button-group.js:57,0]
Side effects in initialization of unused variable label [./~/material-ui/lib/js/radio-button-group.js:57,0]
Side effects in initialization of unused variable onCheck [./~/material-ui/lib/js/radio-button-group.js:57,0]
Side effects in initialization of unused variable Paper [./~/material-ui/lib/js/radio-button-group.js:2,0]
Side effects in initialization of unused variable EnhancedSwitch [./~/material-ui/lib/js/radio-button-group.js:4,0]
Side effects in initialization of unused variable innerClassName [./~/material-ui/lib/js/ripples/circle.js:20,0]
Side effects in initialization of unused variable started [./~/material-ui/lib/js/ripples/circle.js:20,0]
Side effects in initialization of unused variable ending [./~/material-ui/lib/js/ripples/circle.js:20,0]
Side effects in initialization of unused variable value [./~/material-ui/lib/js/slider.js:133,0]
Side effects in initialization of unused variable Paper [./~/material-ui/lib/js/slider.js:2,0]
Side effects in initialization of unused variable Tab [./~/material-ui/lib/js/tabs/tabs.js:2,0]
Side effects in initialization of unused variable length [./~/material-ui/~/react-draggable2/lib/draggable.js:56,0]
Dropping side-effect-free statement [./~/react-tween-state/~/tween-functions/index.js:102,0]
Dropping side-effect-free statement [./~/react-tween-state/~/tween-functions/index.js:101,0]
Dropping side-effect-free statement [./~/react-tween-state/~/tween-functions/index.js:105,0]
Dropping side-effect-free statement [./~/react-tween-state/~/tween-functions/index.js:104,0]
Dropping side-effect-free statement [./~/react-tween-state/~/tween-functions/index.js:136,0]
Dropping side-effect-free statement [./~/react-tween-state/~/tween-functions/index.js:138,0]
Dropping side-effect-free statement [./~/react-tween-state/~/tween-functions/index.js:158,0]
Dropping side-effect-free statement [./~/react-tween-state/~/tween-functions/index.js:160,0]
Dropping side-effect-free statement [./~/react-tween-state/~/tween-functions/index.js:180,0]
Dropping side-effect-free statement [./~/react-tween-state/~/tween-functions/index.js:182,0]
Condition always true [./~/reqwest/reqwest.js:9,0]
Dropping unreachable code [./~/reqwest/reqwest.js:10,0]
Side effects in initialization of unused variable sourceMap [./~/style-loader/addStyles.js:185,0]
Side effects in initialization of unused variable media [./~/style-loader/addStyles.js:203,0]
Condition always false [./~/style-loader/addStyles.js:23,0]
Dropping unreachable code [./~/style-loader/addStyles.js:24,0]
Condition always false [./~/style-loader!./~/css-loader?minimize!./~/postcss-loader!./~/less-loader!./app/styles/main.less:10,0]
Dropping unreachable code [./~/style-loader!./~/css-loader?minimize!./~/postcss-loader!./~/less-loader!./app/styles/main.less:12,0]
Side effects in initialization of unused variable update [./~/style-loader!./~/css-loader?minimize!./~/postcss-loader!./~/less-loader!./app/styles/main.less:7,0]
Child
                                     Asset     Size  Chunks             Chunk Names
      f7c2b4b747b1a225eb8dee034134a1b0.eot  60.8 kB          [emitted]  
      706450d7bba6374ca02fe167d86685cb.ttf   122 kB          [emitted]  
      2980083682e94d33a66eef2e7d612519.svg   313 kB          [emitted]  
    97493d3f11c0a3bd5cbd959f5d19b699.woff2  56.8 kB          [emitted]  
     d9ee23d59d0e0e727b51368b458a0bff.woff  71.5 kB          [emitted]  
                                    app.js   696 kB       0  [emitted]  main
    
    WARNING in ./app/scripts/stores/app-store.js
    
    /home/weave/app/scripts/stores/app-store.js
      177:6  warning  Unexpected console statement  no-console
    
    ✖ 1 problem (0 errors, 1 warning)
    
    
    WARNING in ./app/scripts/charts/nodes-chart.js
    
    /home/weave/app/scripts/charts/nodes-chart.js
      166:12  warning  Unexpected console statement  no-console
      192:4   warning  Unexpected console statement  no-console
      201:4   warning  Unexpected console statement  no-console
    
    ✖ 3 problems (0 errors, 3 warnings)
    
    
    WARNING in ./app/scripts/charts/nodes-layout.js
    
    /home/weave/app/scripts/charts/nodes-layout.js
      12:4  warning  Unexpected console statement  no-console
    
    ✖ 1 problem (0 errors, 1 warning)
    
    
    WARNING in ./~/dagre/lib/graphlib.js
    Critical dependencies:
    5:4-11 require function is used in a way in which dependencies cannot be statically extracted
     @ ./~/dagre/lib/graphlib.js 5:4-11
    
    WARNING in ./~/dagre/lib/lodash.js
    Critical dependencies:
    5:4-11 require function is used in a way in which dependencies cannot be statically extracted
     @ ./~/dagre/lib/lodash.js 5:4-11
    
    WARNING in app.js from UglifyJs
    Side effects in initialization of unused variable className [./~/material-ui/lib/js/paper.js:31,0]
    Side effects in initialization of unused variable circle [./~/material-ui/lib/js/paper.js:31,0]
    Side effects in initialization of unused variable innerClassName [./~/material-ui/lib/js/paper.js:31,0]
    Side effects in initialization of unused variable rounded [./~/material-ui/lib/js/paper.js:31,0]
    Side effects in initialization of unused variable zDepth [./~/material-ui/lib/js/paper.js:31,0]
    Condition always true [./~/dagre/~/graphlib/lib/lodash.js:5,0]
    Side effects in initialization of unused variable support [./~/lodash/index.js:4351,0]
    Dropping unused variable object [./~/lodash/index.js:948,0]
    Condition always true [./~/lodash/index.js:12207,0]
    Dropping unreachable code [./~/lodash/index.js:12221,0]
    Side effects in initialization of unused variable moduleExports [./~/lodash/index.js:275,0]
    Side effects in initialization of unused variable onBlur [./~/material-ui/lib/js/enhanced-button.js:46,0]
    Side effects in initialization of unused variable onFocus [./~/material-ui/lib/js/enhanced-button.js:46,0]
    Side effects in initialization of unused variable onTouchTap [./~/material-ui/lib/js/enhanced-button.js:46,0]
    Side effects in initialization of unused variable className [./~/material-ui/lib/js/font-icon.js:13,0]
    Side effects in initialization of unused variable hasNewDefaultProp [./~/material-ui/lib/js/enhanced-switch.js:59,0]
    Side effects in initialization of unused variable type [./~/material-ui/lib/js/enhanced-switch.js:94,0]
    Side effects in initialization of unused variable name [./~/material-ui/lib/js/enhanced-switch.js:94,0]
    Side effects in initialization of unused variable value [./~/material-ui/lib/js/enhanced-switch.js:94,0]
    Side effects in initialization of unused variable label [./~/material-ui/lib/js/enhanced-switch.js:94,0]
    Side effects in initialization of unused variable onSwitch [./~/material-ui/lib/js/enhanced-switch.js:94,0]
    Side effects in initialization of unused variable defaultSwitched [./~/material-ui/lib/js/enhanced-switch.js:94,0]
    Side effects in initialization of unused variable onBlur [./~/material-ui/lib/js/enhanced-switch.js:94,0]
    Side effects in initialization of unused variable onFocus [./~/material-ui/lib/js/enhanced-switch.js:94,0]
    Side effects in initialization of unused variable onMouseUp [./~/material-ui/lib/js/enhanced-switch.js:94,0]
    Side effects in initialization of unused variable onMouseDown [./~/material-ui/lib/js/enhanced-switch.js:94,0]
    Side effects in initialization of unused variable onMouseOut [./~/material-ui/lib/js/enhanced-switch.js:94,0]
    Side effects in initialization of unused variable onTouchStart [./~/material-ui/lib/js/enhanced-switch.js:94,0]
    Side effects in initialization of unused variable onTouchEnd [./~/material-ui/lib/js/enhanced-switch.js:94,0]
    Side effects in initialization of unused variable icon [./~/material-ui/lib/js/menu/menu.js:195,0]
    Side effects in initialization of unused variable data [./~/material-ui/lib/js/menu/menu.js:195,0]
    Side effects in initialization of unused variable attribute [./~/material-ui/lib/js/menu/menu.js:195,0]
    Side effects in initialization of unused variable number [./~/material-ui/lib/js/menu/menu.js:195,0]
    Side effects in initialization of unused variable toggle [./~/material-ui/lib/js/menu/menu.js:195,0]
    Side effects in initialization of unused variable onClick [./~/material-ui/lib/js/menu/menu.js:195,0]
    Condition always true [./~/d3/d3.js:9502,0]
    Dropping unreachable code [./~/d3/d3.js:9502,61]
    Side effects in initialization of unused variable className [./~/material-ui/lib/js/transition-groups/slide-in.js:24,0]
    Side effects in initialization of unused variable direction [./~/material-ui/lib/js/transition-groups/slide-in.js:24,0]
    Dropping unused variable isCurryBound [./~/dagre/~/lodash/dist/lodash.compat.js:1674,0]
    Condition always true [./~/dagre/~/lodash/dist/lodash.compat.js:7131,0]
    Dropping unreachable code [./~/dagre/~/lodash/dist/lodash.compat.js:7144,0]
    Side effects in initialization of unused variable moduleExports [./~/dagre/~/lodash/dist/lodash.compat.js:175,0]
    Side effects in initialization of unused variable onClick [./~/material-ui/lib/js/menu/menu-item.js:68,0]
    Side effects in initialization of unused variable onToggle [./~/material-ui/lib/js/menu/menu-item.js:68,0]
    Side effects in initialization of unused variable children [./~/material-ui/lib/js/menu/menu-item.js:68,0]
    Side effects in initialization of unused variable label [./~/material-ui/lib/js/menu/menu-item.js:68,0]
    Side effects in initialization of unused variable className [./~/material-ui/lib/js/overlay.js:28,0]
    Side effects in initialization of unused variable onCheck [./~/material-ui/lib/js/radio-button.js:20,0]
    Side effects in initialization of unused variable TabTemplate [./~/material-ui/lib/js/tabs/tab.js:3,0]
    Side effects in initialization of unused variable className [./~/material-ui/lib/js/text-field.js:70,0]
    Side effects in initialization of unused variable errorText [./~/material-ui/lib/js/text-field.js:70,0]
    Side effects in initialization of unused variable floatingLabelText [./~/material-ui/lib/js/text-field.js:70,0]
    Side effects in initialization of unused variable hintText [./~/material-ui/lib/js/text-field.js:70,0]
    Side effects in initialization of unused variable id [./~/material-ui/lib/js/text-field.js:70,0]
    Side effects in initialization of unused variable multiLine [./~/material-ui/lib/js/text-field.js:70,0]
    Side effects in initialization of unused variable onBlur [./~/material-ui/lib/js/text-field.js:70,0]
    Side effects in initialization of unused variable onChange [./~/material-ui/lib/js/text-field.js:70,0]
    Side effects in initialization of unused variable onFocus [./~/material-ui/lib/js/text-field.js:70,0]
    Side effects in initialization of unused variable type [./~/material-ui/lib/js/text-field.js:70,0]
    Side effects in initialization of unused variable onToggle [./~/material-ui/lib/js/toggle.js:20,0]
    Side effects in initialization of unused variable className [./~/material-ui/lib/js/tooltip.js:27,0]
    Side effects in initialization of unused variable label [./~/material-ui/lib/js/tooltip.js:27,0]
    Condition always true [./~/material-ui/~/classnames/index.js:39,0]
    Condition always false [./~/flux/lib/invariant.js:26,0]
    Dropping unreachable code [./~/flux/lib/invariant.js:27,0]
    Condition always false [./~/font-awesome-webpack/~/style-loader/addStyles.js:23,0]
    Dropping unreachable code [./~/font-awesome-webpack/~/style-loader/addStyles.js:24,0]
    Condition always false [./~/font-awesome-webpack/~/style-loader!./~/font-awesome-webpack/~/css-loader!./~/font-awesome-webpack/~/less-loader!./~/font-awesome-webpack/font-awesome-styles.loader.js!./~/font-awesome-webpack/font-awesome.config.js:9,0]
    Dropping unreachable code [./~/font-awesome-webpack/~/style-loader!./~/font-awesome-webpack/~/css-loader!./~/font-awesome-webpack/~/less-loader!./~/font-awesome-webpack/font-awesome-styles.loader.js!./~/font-awesome-webpack/font-awesome.config.js:11,0]
    Side effects in initialization of unused variable update [./~/font-awesome-webpack/~/style-loader!./~/font-awesome-webpack/~/css-loader!./~/font-awesome-webpack/~/less-loader!./~/font-awesome-webpack/font-awesome-styles.loader.js!./~/font-awesome-webpack/font-awesome.config.js:7,0]
    Side effects in initialization of unused variable onTouchTap [./~/material-ui/lib/js/app-bar.js:42,0]
    Side effects in initialization of unused variable other [./~/material-ui/lib/js/app-bar.js:42,0]
    Side effects in initialization of unused variable onCheck [./~/material-ui/lib/js/checkbox.js:19,0]
    Dropping unused variable newSelectedDate [./~/material-ui/lib/js/date-picker/calendar.js:159,0]
    Side effects in initialization of unused variable selectedDate [./~/material-ui/lib/js/date-picker/date-display.js:35,0]
    Side effects in initialization of unused variable initialDate [./~/material-ui/lib/js/date-picker/date-picker-dialog.js:37,0]
    Side effects in initialization of unused variable onAccept [./~/material-ui/lib/js/date-picker/date-picker-dialog.js:37,0]
    Side effects in initialization of unused variable formatDate [./~/material-ui/lib/js/date-picker/date-picker.js:59,0]
    Side effects in initialization of unused variable mode [./~/material-ui/lib/js/date-picker/date-picker.js:59,0]
    Side effects in initialization of unused variable onFocus [./~/material-ui/lib/js/date-picker/date-picker.js:59,0]
    Side effects in initialization of unused variable onTouchTap [./~/material-ui/lib/js/date-picker/date-picker.js:59,0]
    Side effects in initialization of unused variable KeyCode [./~/material-ui/lib/js/date-picker/date-picker.js:5,0]
    Side effects in initialization of unused variable className [./~/material-ui/lib/js/date-picker/day-button.js:23,0]
    Side effects in initialization of unused variable date [./~/material-ui/lib/js/date-picker/day-button.js:23,0]
    Side effects in initialization of unused variable onTouchTap [./~/material-ui/lib/js/date-picker/day-button.js:23,0]
    Side effects in initialization of unused variable selected [./~/material-ui/lib/js/date-picker/day-button.js:23,0]
    Side effects in initialization of unused variable className [./~/material-ui/lib/js/dialog.js:17,0]
    Side effects in initialization of unused variable KeyLine [./~/material-ui/lib/js/drop-down-icon.js:4,0]
    Side effects in initialization of unused variable Paper [./~/material-ui/lib/js/drop-down-icon.js:5,0]
    Side effects in initialization of unused variable className [./~/material-ui/lib/js/enhanced-textarea.js:41,0]
    Side effects in initialization of unused variable onChange [./~/material-ui/lib/js/enhanced-textarea.js:41,0]
    Side effects in initialization of unused variable onHeightChange [./~/material-ui/lib/js/enhanced-textarea.js:41,0]
    Side effects in initialization of unused variable rows [./~/material-ui/lib/js/enhanced-textarea.js:41,0]
    Side effects in initialization of unused variable valueLink [./~/material-ui/lib/js/enhanced-textarea.js:41,0]
    Side effects in initialization of unused variable name [./~/material-ui/lib/js/radio-button-group.js:57,0]
    Side effects in initialization of unused variable value [./~/material-ui/lib/js/radio-button-group.js:57,0]
    Side effects in initialization of unused variable label [./~/material-ui/lib/js/radio-button-group.js:57,0]
    Side effects in initialization of unused variable onCheck [./~/material-ui/lib/js/radio-button-group.js:57,0]
    Side effects in initialization of unused variable Paper [./~/material-ui/lib/js/radio-button-group.js:2,0]
    Side effects in initialization of unused variable EnhancedSwitch [./~/material-ui/lib/js/radio-button-group.js:4,0]
    Side effects in initialization of unused variable innerClassName [./~/material-ui/lib/js/ripples/circle.js:20,0]
    Side effects in initialization of unused variable started [./~/material-ui/lib/js/ripples/circle.js:20,0]
    Side effects in initialization of unused variable ending [./~/material-ui/lib/js/ripples/circle.js:20,0]
    Side effects in initialization of unused variable value [./~/material-ui/lib/js/slider.js:133,0]
    Side effects in initialization of unused variable Paper [./~/material-ui/lib/js/slider.js:2,0]
    Side effects in initialization of unused variable Tab [./~/material-ui/lib/js/tabs/tabs.js:2,0]
    Side effects in initialization of unused variable length [./~/material-ui/~/react-draggable2/lib/draggable.js:56,0]
    Dropping side-effect-free statement [./~/react-tween-state/~/tween-functions/index.js:102,0]
    Dropping side-effect-free statement [./~/react-tween-state/~/tween-functions/index.js:101,0]
    Dropping side-effect-free statement [./~/react-tween-state/~/tween-functions/index.js:105,0]
    Dropping side-effect-free statement [./~/react-tween-state/~/tween-functions/index.js:104,0]
    Dropping side-effect-free statement [./~/react-tween-state/~/tween-functions/index.js:136,0]
    Dropping side-effect-free statement [./~/react-tween-state/~/tween-functions/index.js:138,0]
    Dropping side-effect-free statement [./~/react-tween-state/~/tween-functions/index.js:158,0]
    Dropping side-effect-free statement [./~/react-tween-state/~/tween-functions/index.js:160,0]
    Dropping side-effect-free statement [./~/react-tween-state/~/tween-functions/index.js:180,0]
    Dropping side-effect-free statement [./~/react-tween-state/~/tween-functions/index.js:182,0]
    Condition always true [./~/reqwest/reqwest.js:9,0]
    Dropping unreachable code [./~/reqwest/reqwest.js:10,0]
    Side effects in initialization of unused variable sourceMap [./~/style-loader/addStyles.js:185,0]
    Side effects in initialization of unused variable media [./~/style-loader/addStyles.js:203,0]
    Condition always false [./~/style-loader/addStyles.js:23,0]
    Dropping unreachable code [./~/style-loader/addStyles.js:24,0]
    Condition always false [./~/style-loader!./~/css-loader?minimize!./~/postcss-loader!./~/less-loader!./app/styles/main.less:10,0]
    Dropping unreachable code [./~/style-loader!./~/css-loader?minimize!./~/postcss-loader!./~/less-loader!./app/styles/main.less:12,0]
    Side effects in initialization of unused variable update [./~/style-loader!./~/css-loader?minimize!./~/postcss-loader!./~/less-loader!./app/styles/main.less:7,0]
[12:01:30] Finished 'bundle' after 23 s
[12:01:30] Finished 'build' after 24 s
esc -o app/static.go -prefix client/build client/build
```

--- a/client/app/scripts/charts/nodes-chart.js
+++ b/client/app/scripts/charts/nodes-chart.js
@@ -1,6 +1,8 @@
 const _ = require('lodash');
 const d3 = require('d3');
+const debug = require('debug')('nodes-chart');
 const React = require('react');
+const timely = require('timely');
 
 const Edge = require('./edge');
 const Naming = require('../constants/naming');
@@ -163,7 +165,7 @@ const NodesChart = React.createClass({
           const target = nodes[edge[1]];
 
           if (!source || !target) {
-            console.error('Missing edge node', edge[0], source, edge[1], target);
+            debug('Missing edge node', edge[0], source, edge[1], target);
           }
 
           edges[edgeId] = {
@@ -188,9 +190,8 @@ const NodesChart = React.createClass({
     const n = _.size(props.nodes);
     const nodeScale = d3.scale.linear().range([0, nodeSize / Math.pow(n, 0.7)]);
 
-    const layoutId = 'layered node chart';
-    console.time(layoutId);
-    const graph = NodesLayout.doLayout(
+    const timedLayouter = timely(NodesLayout.doLayout);
+    const graph = timedLayouter(
       nodes,
       edges,
       props.width,
@@ -198,7 +199,8 @@ const NodesChart = React.createClass({
       nodeScale,
       MARGINS
     );
-    console.timeEnd(layoutId);
+
+    debug('graph layout took ' + timedLayouter.time + 'ms');
 
     // adjust layout based on viewport
 

--- a/client/app/scripts/charts/nodes-layout.js
+++ b/client/app/scripts/charts/nodes-layout.js
@@ -1,4 +1,5 @@
 const dagre = require('dagre');
+const debug = require('debug')('nodes-layout');
 const _ = require('lodash');
 
 const MAX_NODES = 100;
@@ -9,7 +10,7 @@ const doLayout = function(nodes, edges, width, height, scale, margins) {
   const g = new dagre.graphlib.Graph({});
 
   if (_.size(nodes) > MAX_NODES) {
-    console.error('Too many nodes for graph layout engine. Limit: ' + MAX_NODES);
+    debug('Too many nodes for graph layout engine. Limit: ' + MAX_NODES);
     return null;
   }
 

--- a/client/app/scripts/stores/app-store.js
+++ b/client/app/scripts/stores/app-store.js
@@ -1,6 +1,7 @@
 const EventEmitter = require('events').EventEmitter;
 const _ = require('lodash');
 const assign = require('object-assign');
+const debug = require('debug')('app-store');
 
 const AppDispatcher = require('../dispatcher/app-dispatcher');
 const ActionTypes = require('../constants/action-types');
@@ -179,7 +180,7 @@ AppStore.registeredCallback = function(payload) {
       break;
 
     case ActionTypes.RECEIVE_NODES_DELTA:
-      console.log('RECEIVE_NODES_DELTA',
+      debug('RECEIVE_NODES_DELTA',
         'remove', _.size(payload.delta.remove),
         'update', _.size(payload.delta.update),
         'add', _.size(payload.delta.add));

--- a/client/package.json
+++ b/client/package.json
@@ -8,6 +8,7 @@
   "dependencies": {
     "d3": "^3.5.5",
     "dagre": "^0.7.2",
+    "debug": "^2.2.0",
     "flux": "^2.0.3",
     "font-awesome": "^4.3.0",
     "keymirror": "^0.1.1",
@@ -18,7 +19,8 @@
     "react": "^0.13.3",
     "react-tap-event-plugin": "^0.1.7",
     "react-tween-state": "0.0.5",
-    "reqwest": "~1.1.5"
+    "reqwest": "~1.1.5",
+    "timely": "^0.1.0"
   },
   "devDependencies": {
     "autoprefixer-core": "^5.2.0",

--- a/client/webpack.config.js
+++ b/client/webpack.config.js
@@ -116,7 +116,7 @@ var appConfig = _.merge({}, config, {
   },
   plugins: config.plugins.concat(DEBUG ? [] : [
       new webpack.optimize.DedupePlugin(),
-      new webpack.optimize.UglifyJsPlugin(),
+      new webpack.optimize.UglifyJsPlugin({compress: {warnings: false}}),
       new webpack.optimize.AggressiveMergingPlugin()
     ]
   )

